mantidproject/mantid
# update larmor idf

Closes #13682 

**Context**

Two new IDF files were added for the LARMOR instrument, so have three versions in total. They are
- IDF1: LARMOR_Definition_19000000-20150317.xml which is valid from 1900-00-00 to 2015-03-17
- IDF2: LARMOR_Definition_8tubes.xml which is valid from "2015-03-18 00:00:00" to "2015-04-29 11:59:59"
- IDF2: LARMOR_Definition.xml which is valid from "2015-04-29 12:00:00" to end-of-time 
### For testing:

We want to make sure that the correct IDF is selected for a workspace which was generated in one of the three time slots. For each time slot there is a worspace which can be found here:
- IDF1: \olympic\Babylon5\Scratch\Anton\Testing\13682_update_larmor_idf\IDF1
- IDF2: \olympic\Babylon5\Scratch\Anton\Testing\13682_update_larmor_idf\IDF2
- IDF3: \olympic\Babylon5\Scratch\Anton\Testing\13682_update_larmor_idf\IDF3

**Test that correct IDF is loaded**
Do this for each data set for IDF1, IDF2, IDF3
1. Load the data set into Mantid
2. In Mantid Dock check the IDF entry.
   - Confirm that the workspace contains the matching IDF (you should see a string with the name of the IDF)
